suites/quincy/e2e/poc/test_cluster_deployment.yaml

```diff
@@ -245,12 +246,12 @@ tests:
       abort-on-fail: false
       config:
         haproxy_clients:
-          - cali019
+          - argo018
         rgw_endpoints:
-          - cali015:80
-          - cali016:80
-          - cali017:80
-          - cali018:80
+          - argo027:80
```

Comment:
same as above

Reply:
updated and executed 
Logs : https://jenkins.ceph.redhat.com/view/RHCS%20QE/job/rhceph-upi-pipeline-executor/36/console